src/containers/LearningpathPage/LearningpathPage.tsx

```diff
@@ -204,10 +204,13 @@ const getTitle = (
   ]);
 };
 
-const getDocumentTitle = (t: TFunction, data: PropData) => {
+const getDocumentTitle = (t: TFunction, data: PropData, stepId?: string) => {
   const subject = data.subject;
   const learningpath = data.resource?.learningpath;
-  return htmlTitle(getTitle(subject, learningpath), [
+  const step = stepId
+    ? learningpath?.learningsteps?.[parseInt(stepId) - 1]
```

Comment:
Er ikkje stepId et løpenummer på tvers av alle steg? Vil dette fungere for alle stier?

Reply:
Åja, det var jo litt upraktisk. Skal få endret på det